napari/_qt/qt_viewer.py

```diff
@@ -626,8 +626,12 @@ def _save_layers_dialog(self, selected=False):
             ),
         )
         logging.debug(
-            f'QFileDialog - filename: {filename or None} '
-            f'selected_filter: {selected_filter or None}'
+            trans._(
+                'QFileDialog - filename: {filename} '
+                'selected_filter: {selected_filter}',
+                filename=filename or None,
+                selected_filter=selected_filter or None,
+            )
```

Comment:
If I correctly remember the trans should consume only single-line strings. Also, I'm not sure if debug messages should be translated.

Reply:
I don't think that is true for single line, I believe I fixed this already in #4588

Reply:
WRT to if this has this to be translated, I don't really care. Happy to revert as well.

Reply:
#4588 should be more advertised. Why do I miss it.

Reply:
I think that's because not working on join strings was just a bug, and bug just get fixed :-)